integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -174,19 +191,39 @@ public void lockBlockTest4() throws IOException {
       fileIds.add(TachyonFSTestUtils.createByteFile(mTfs, uniqPath + k, WriteType.CACHE_THROUGH,
           fileSize));
     }
+
+    // TODO: test cache stream, since file.recache() is to be removed
     for (int k = 0; k <= numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
-      CommonUtils.sleepMs(null, getSleepMs());
       Assert.assertFalse(tFile.isInMemory());
+      is = tFile.getInStream(ReadType.NO_CACHE);
+      buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
+      CommonUtils.sleepMs(null, getSleepMs());
+      if (is.markSupported()) {
```

Comment:
A little bit confused by the documentation. But since it says "The markSupported method of InputStream returns false." InStream's markSupported will return false. Thanks, will change this to using two separate string to read.